cloudinit/config/cc_resizefs.py

```diff
@@ -60,15 +60,28 @@ def _resize_btrfs(mount_point, devpth):
     if not util.mount_is_read_write(mount_point) and os.path.isdir(
         "%s/.snapshots" % mount_point
     ):
-        return (
+        cmd = [
             "btrfs",
             "filesystem",
             "resize",
             "max",
             "%s/.snapshots" % mount_point,
-        )
+        ]
     else:
-        return ("btrfs", "filesystem", "resize", "max", mount_point)
+        cmd = ["btrfs", "filesystem", "resize", "max", mount_point]
+
+    # btrfs has exclusive operations and resize may fail if btrfs is busy
+    # doing one of the operations that prevents resize. As of btrfs 5.10
+    # the resize operation can be queued
+    btrfs_with_queue = util.Version().from_str("5.10")
```

Comment:
fyi: Since `from_str()` is a class method, creating Version object instances isn't necessary. It could be this instead:
```python
    btrfs_with_queue = util.Version.from_str("5.10")
```

This line and the next would save us a couple of object instantiations, which I would expect to be insignificant but measurable.